specs/refactor_common_code_and_move.adoc

```diff
@@ -203,4 +232,8 @@ None
 
 == References
 
-None
+* https://github.com/Tendrl/common/issues/58
+
+* https://github.com/Tendrl/common/issues/60
+
+* https://github.com/Tendrl/common/issues/62
```

Comment:
@shtripat can you please include: https://github.com/Tendrl/common/issues/80. Sorry for informing you late.